### ISSUE TITLE
TypeScript => possibility of integrating this into NativeScript?

### ISSUE THREAD

**Issue by thenameisdavid** [tsedio/ts-gphoto2-driver#16]
Nice job putting this together in TypeScript! So, I have a question (sorry if I'm not adhering to proper Github etiquette here):

TLDR: how difficult would it be to port this over into an iOS project? 

There are several threads in the Gphoto universe which, previously, questioned whether incorporating Gphoto's functionality into XCode/iOS projects would be feasible. The Gphoto devs seemed to think it wouldn't work -- for Apple's AppStore distribution purposes -- due to the use of private and/or low-level frameworks which weren't accessible or permitted. However; these discussions also took place several years ago, and the Apple eco system is constantly evolving, so I'm wondering how feasible it would be to do this today. 

Also, given that NativeScript allows the usage of TypeScript, how far fetched of a possibility would it be to port this into NativeScript for iOS purposes -- for private (less restrictions on library use) or public distribution?

I know we could just create a WebView connecting to a local server running on a desktop or Android phone, and then use CGI or other scripts to trigger the functionality. I'm curious as to how we could pull this off running it as a standalone app without external server requirements. 

Cheers!

**Comment by Romakita** [comment]
Hello @thenameisdavid 
I'm not sure if it's possible. The typescript (or javascript) version use C++ node.js extension to bind the native driver. The gphoto driver should be installed with the right linux/mac command line. Also, the driver must support the targeted platform. I'm not sure if Gphoto driver is design for iOS or Android. 
The question should be ask to Gphoto driver team :)

See you
Romain
